server/src/main/java/com/adobe/testing/s3mock/domain/FileStore.java

```diff
@@ -1041,10 +1045,10 @@ private File retrieveFile(final String bucketName, final String fileName, final
     return parts;
   }
 
-  private String calculateHashOfFilePart(final File currentFilePart, final int partNumber) {
+  private String calculateHashOfFilePart(final File currentFilePart) {
     try (final InputStream is = FileUtils.openInputStream(currentFilePart)) {
       final String partMd5 = DigestUtils.md5Hex(is);
-      return String.format("%s-%s", partMd5, partNumber);
+      return String.format("%s", partMd5);
```

Comment:
Verified that part number is not appended for each individual part e-tag, it is appended only for the e-tag of the complete file that is constructed from parts.